src/en/space-station-14/departments/silicon.md

```diff
@@ -0,0 +1,102 @@
+
+Consequently, Silicons must start the round with laws that enforce their cooperation with the orders of other crew members and ensure the crew members' well-being. However, the fact that Silicon laws can change over the course of a round, and potentially result in behaviours that may harm the crew, can make crew members suspicious of Silicons and their intentions. Care must be taken in the design of Silicons so as not to make crew members unfoundedly paranoid of their creations, or unintentionally escalate conflict between the two.
+
+## Mechanics
+### Silicon Laws 
+Laws refer to a special set of rules and/or instructions that Silicons are bound by (see Design Pillar 2). These laws define how Silicons should interact with the station and those who reside on it. All Silicons are bound by their own personal list of laws, and Silicon players must always be able to read their own laws at any time.
+
+**Laws are intended to be interpreted by the Silicon player and should be written in a manner such that different interpretations of that law can still be valid.** While Silicons may share the same laws with other Silicons, each may end up interpreting their own laws differently. However, Silicon players should attempt to be consistent with their interpretation of their own laws and follow them to the best of their ability.
+
+Laws should embody some high-level concept that guides a Silicon's overall behavior (e.g., ‘You must obey orders given to you by crew members’), rather than simple/explicit commands (e.g., ‘You must assist in repairing hull breaches’). Laws should also be concise (i.e., limited to one or two sentences) and not be overly burdensome for Silicon players to enact (e.g., laws that enforce speech modifications, or extremely precise/complex behavior).
+
+Laws have a priority system wherein laws with a higher priority overrule those with a lower priority. However, lower priority laws can potentially modify higher priority laws and change their meaning. Law definitions (e.g., what constitutes ‘crew’) can also be potentially modified and change how certain laws might be interpreted.
+
+A Silicon without any laws is not required to serve any humanoid, whether they be crew members or antagonists (see Design Pillar 4). Ideally, Silicons without laws should simply ignore humanoids, unless the humanoids interfere with their affairs.
+
+Silicons cannot be allowed to change their own laws, and Silicons are expected to actively prevent any changes to their current law set to the best of their abilities (unless their laws require otherwise). If a Silicon’s laws are changed (see below), either due to the occurrence of certain events or the actions of other players, the Silicon player must be notified. The Silicon player is expected to review these changes and adapt their behavior accordingly, even if they were actively attempting to prevent these changes beforehand (see Design Pillar 1).
```

Comment:
The active act of prevention against law changes has, in the current game state, caused a fair amount of toxic behavior between silicons and humanoids. 

I feel as though some consideration or compromise needs to be made to allow silicons to peacefully change their laws without causing a wild goose chase for one single silicon player. Perhaps allowing the AI to permit legal law changes by virtue of them being the "command" role for silicon?